samples/oauth2/sparklr/pom.xml

```diff
@@ -157,6 +157,12 @@
 			<version>${jackson2.version}</version>
 		</dependency>
 
+   <dependency>
```

Comment:
The `POM` already includes:

```
		<dependency>
			<groupId>com.fasterxml.jackson.core</groupId>
			<artifactId>jackson-databind</artifactId>
			<version>${jackson2.version}</version>
		</dependency>
```

Which transitively imports `com.fasterxml.jackson.core:jackson-core`.

So I don't see why this change is needed?